omod/src/main/webapp/fragments/tags.gsp

```diff
@@ -0,0 +1,58 @@
+<%
+    ui.includeJavascript("uicommons", "angular.min.js")
```

Comment:
Can you confirm if some of these includes aren't already included on the page, i.e remove them and see if it still works.